### PR TITLE
Improve doc install pages

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR builds on #3066 focusing on the install pages, to make it easier to access the various methods we have for installation, the index page is updated with links and nestml and nest-desktop are clearly marked as well as projects to look at as well.
It cannot be merged until #3066 so will be labeled blocked

### CODE REVIEW

**Comment by jessica-mitchell** [comment]
@ddahmen @heplesser @steffengraber @clinssen 

[output here](https://nest-simulator--3067.org.readthedocs.build/en/3067/)

Changes made

- [x] removed HPC card and instructions
- [x] moved minimal configuration to cmake options
- [x] Add EBRAINS install free in separate PR

TODO 

improve HPC install instructions

File: doc/htmldoc/static/img/arrow_left.svg
Comment (clinssen) [inline doc/htmldoc/static/img/arrow_left.svg:1]: this svg has an artefact in the top right corner!

Comment (jessica-mitchell) [reply doc/htmldoc/static/img/arrow_left.svg:1]: Ah actually in one of these PRs, this svg is deleted.

**Review by heplesser — APPROVED** [review]
@jessica-mitchell Looks good to me! I suggest to create a new issue to remind us of the need to provide better HPC installation instructions.

**Review by heplesser — CHANGES_REQUESTED** [review]
Actually, I noticed that in the #3066 version, we have page https://nest-simulator--3066.org.readthedocs.build/en/3066/installation/admin.html#admin-install. On it, the link under "Configure HPC Systems" leads to a page without useful information, but the other links, namely 
- https://nest-simulator--3066.org.readthedocs.build/en/3066/hpc/optimizing_nest.html#optimize-performance
- links to docker instructions
provide useful information. It would be good if we could keep links to them from the install page.

**Comment by jessica-mitchell** [comment]
> Actually, I noticed that in the #3066 version, we have page https://nest-simulator--3066.org.readthedocs.build/en/3066/installation/admin.html#admin-install. On it, the link under "Configure HPC Systems" leads to a page without useful information, but the other links, namely
> 
>     * https://nest-simulator--3066.org.readthedocs.build/en/3066/hpc/optimizing_nest.html#optimize-performance
> 
>     * links to docker instructions
>       provide useful information. It would be good if we could keep links to them from the install page.

So the docker instructions are already on the install page, but I added a card about configuring HPC systems there

**Review by heplesser — COMMENTED** [review]
@jessica-mitchell Would you mind moving the "Configure HPC" card to the top row? It looks rather strange with the open space on the top right.

**Comment by heplesser** [comment]
> > @jessica-mitchell Would you mind moving the "Configure HPC" card to the top row? It looks rather strange with the open space on the top right.
> 
> This will be fixed in the PR that adds the EBRAINS card #3121

Good!

**Review by heplesser — APPROVED** [review]

File: doc/htmldoc/installation/index.rst [hunk 1]
```diff
@@ -3,40 +3,63 @@
...
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
-    .. grid-item-card:: |dev| Install NEST from source
-       :class-title: sd-d-flex-row sd-align-minor-center
+      A graphical user interface, ideal for learning and teaching concepts
+      regarding neural networks in classrooms and workshops.
 
-       I'm a developer who wants to :ref:`do development in NEST <dev_install>`
+      :doc:`NEST Desktop documentation <desktop:index>` :octicon:`link-external`
```
Comment (clinssen) [inline doc/htmldoc/installation/index.rst:40]: I would suggest to use bullet points in all the cards for consistency.

**Review by clinssen — COMMENTED** [review]
Thanks for the updates! I would suggest to add a subheader for "Install related tools" or such for NEST Desktop and NESTML, to prevent confusion.

File: doc/htmldoc/installation/index.rst (hunk 1 above)
Comment (jessica-mitchell) [reply doc/htmldoc/installation/index.rst]: Ok I updated the text.

Comment (jessica-mitchell) [reply doc/htmldoc/installation/index.rst]: @clinssen what do you think now?

**Review by clinssen — APPROVED** [review]

**Pull request merged**